.github/workflows/build.yml

```diff
@@ -0,0 +1,38 @@
+name: Go
+on: [push]
+jobs:
+  run:
+    runs-on: ${{ matrix.os }}
+    strategy:
+      matrix:
+        os: [ubuntu-latest]
+    env:
+      OS: ${{ matrix.os }}
+    steps:
+      - name: Set up Go
+        uses: actions/setup-go@v2
+        id: go
+
+      - name: Check out code into the Go module directory
+        uses: actions/checkout@v1
+
+      - name: Get dependencies
+        run: |
+          go get -v -t -d ./...
+
+      - name: Build compliance-masonry
+        run: go build cmd/compliance-masonry/compliance-masonry.go
```

Comment:
this ought to work just fine; we might benefit from recycling the tooling paths for consistency-- by that I mean, we should be able to get away with doing `make build` here based on the Makefile (https://github.com/opencontrol/compliance-masonry/blob/master/Makefile), or by doing `goreleaser build` (assuming the `goreleaser` binary was made available to the host); it all does the same thing, but it's through the same 'build path' so it's more concise. just a suggestion in any case  👍  this should still work